libs/src/sdk/api/users/UsersApi.ts

```diff
@@ -15,21 +18,88 @@ import {
   FollowUserSchema,
   SubscribeToUserRequest,
   SubscribeToUserSchema,
+  UpdateProfileRequest,
   UnfollowUserRequest,
   UnfollowUserSchema,
   UnsubscribeFromUserRequest,
-  UnsubscribeFromUserSchema
+  UnsubscribeFromUserSchema,
+  UpdateProfileSchema
 } from './types'
 
 export class UsersApi extends GeneratedUsersApi {
   constructor(
     configuration: Configuration,
+    private readonly storage: StorageService,
     private readonly entityManager: EntityManagerService,
     private readonly auth: AuthService
   ) {
     super(configuration)
   }
 
+  async updateProfile(
```

Comment:
Can you add a quick jsdoc comment, idk how useful they are but we have them for the other methods